src/bp-forums/forums/template.php

```diff
@@ -2740,6 +2795,42 @@ function bbp_get_form_forum_visibility_dropdown( $args = '' ) {
 	return apply_filters( 'bbp_get_form_forum_type_dropdown', ob_get_clean(), $r );
 }
 
+/**
+ * Checking whether you can update or not forum visibility.
+ *
+ * When Forum associated with Group then Forum and all level child Forums "Visibility" Field should be disabled.
+ *
+ * @since bbPress (r3563)
```

Comment:
@asaquzzamanbb why we have this `@since bbPress (r3563)` is this function forked from bbpress?

Reply:
Change `@since BuddyBoss 1.5.9`
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/da785ba8a0338d2bfbd7d4c829c6d554dfa69f8e, 
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/5890058f4ff3e80122785d2eb17b7dd25403a036,
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/114240b580109fa63811374aa0a1f1d1f050cf6f